bears/general/CPDBear.py

```diff
@@ -0,0 +1,40 @@
+import re
+import os
+
+from coalib.bearlib.abstractions.Lint import Lint
+from coalib.bears.LocalBear import LocalBear
+
+
+class CPDBear(LocalBear, Lint):
+    executable = 'run.sh'
```

Comment:
how do you get this? i don't see any dependencies as i would like to try it myself now.